master/internal/api_runs.go

```diff
@@ -239,3 +249,133 @@ func sortRuns(sortString *string, runQuery *bun.SelectQuery) error {
 	}
 	return nil
 }
+
+func (a *apiServer) MoveRuns(
+	ctx context.Context, req *apiv1.MoveRunsRequest,
+) (*apiv1.MoveRunsResponse, error) {
+	curUser, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+	if curUser == nil {
```

Comment:
non-blocking question: is it possible for a user to not be found an `GetUser` to not return an error?

Reply:
Not sure. I looked at other uses of GetUser and they don't seem to do this check. I think I should remove it.